Libraries/vendor/tinycolor/tinycolor.js

```diff
@@ -1,503 +1,341 @@
 /**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
  * @providesModule tinycolor
- * @nolint
+ * @flow
  */
+/* eslint no-bitwise: 0 */
 'use strict';
 
-// TinyColor v1.2.1
-// https://github.com/bgrins/TinyColor
-// Brian Grinstead, MIT License
-
-var trimLeft = /^[\s,#]+/,
-    trimRight = /\s+$/,
-    tinyCounter = 0,
-    mathRound = Math.round,
-    mathMin = Math.min,
-    mathMax = Math.max;
-
-function tinycolor (color, opts) {
-    // If we are called as a function, call using new instead
-    if (!(this instanceof tinycolor)) {
-        return new tinycolor(color, opts);
+function tinycolor(color: string | number): number | null {
+  if (typeof color === 'number') {
+    if (color !== (color >>> 0) || color < 0 || color > 0xffffffff) {
```

Comment:
Previously there was no check to make sure that the number wasn't outside of the acceptable range and not a floating point